project-reports/2023/2023-Q4-Hyperledger-FireFly.md

```diff
@@ -0,0 +1,117 @@
+
+### Community adoption updates
+
+- [Warner Music Group](https://www.wmgrewards.com/)
+  - Warner Music Group rewards is powered by FireFly and Tezos and is live in production
+  - Tokens for sweepstakes are earned by users who link their Spotify account and follow bands and favorite songs
+- Blockchain for Energy Commodity Transport
+  - Blockchain for Energy is a consortium with members including Chevron, ExxonMobil, Respol, Worley, Pioneer, Schlumberger, and more.
+  - The Commodity Transport use case digitizes the current manual validation processes for commodity deliveries and logistics and utilizes FireFly to link together on and off-chain systems.
+  - The application creates real-time visibility and hyper-automation for field logistics, ensuring significant cost reductions, streamlined operations, and frictionless consumer-supplier interactions.
+
+
+Community members have given talks featuring Hyperledger FireFly at a number of events including:
+
+- Hyperledger Webinar: [Driving Digital Transformation in Government with Hyperledger FireFly](https://www.youtube.com/watch?v=M0kfnZzk8jU)
+- Hyperledger Meetup: [Creating and Open Sourcing a Tezos Connector for Hyperledger FireFly](https://www.youtube.com/watch?v=5Y2Wnld0aJg)
+- Sibos 2023: [Managing Digital Assets at Scale: Lessons Learned and What’s Next](https://www.kaleido.io/resources/managing-digital-assets-at-scale-lessons-learned-and-whats-next)
+- Kaleido & AWS event: [Unlocking the Potential of Asset Tokenization](https://www.kaleido.io/nyc-tokenization-event)
+- All Things Open: [An Introduction to Open Source Tools for Building in Web3](https://2023.allthingsopen.org/sessions/an-introduction-to-open-source-tools-for-building-in-web3/)
+- Hyperledger Member Webinar: [The Emerging Tech Stack for Digital Assets and Asset Tokenization](https://www.youtube.com/watch?v=heVWZZnz-j4)
+
+### Contributor activity updates
+
+The star activity for the FireFly repository has continued to grow from 390 end of last quarter to 430 end of this quarter. Discord activity remains consistent with ~45 DAU.
+
+![alt_text](images/HLFF_2023_Q3_image1.png "image_tooltip")
```

Comment:
Assuming this needs to be changes to a Q4 image.

Reply:
@aznrayizzle : Can you please have a look at this change?

Reply:
@aznrayizzle : Can you please update the document based on my feedback?

Reply:
@tkuhrt updated the image. Sorry about the delay in doing so.